Gemfile

```diff
@@ -23,6 +23,7 @@ group :test do
   else
     gem "escape_utils",     "~> 1.0",   :require => false
     gem "github-linguist",  "~> 2.10",  :require => false
+    gem "pygments.rb",                  :require => false
```

Comment:
`github-linguist < 4` don't need to include this dependency, right?

How about we add a comment to inform this?

```ruby
# github-linguist version > 4 need to explicitly include pygments.rb,
# see https://github.com/jch/html-pipeline/issues/235
# gem "pygments.rb",                  :require => false
```